src/status_im/ui/components/colors.cljs

```diff
@@ -12,6 +12,7 @@
 (def gray-light "#e8ebec") ;; Used as divider color
 (def gray-lighter "#eef2f5")  ;; Used as a background or shadow
 (def gray-transparent "rgba(184, 193, 199, 0.5)") ;; Used for tabs
+(def gray-transparent-2 "#4a5054e6") ;; Used for tooltip background
```

Comment:
Do we really need a new gray for this? If yes, can we have a more descriptive name, e.g. `gray-light-transparent` or something? I'd prefer if we don't recreate the same colors list we had in `component.styles`

Reply:
agree @goranjovic , @denis-sharypin  wdyt?

Reply:
Unless there is a big difference I would just use `gray-transparent`.